src/values/ObjectValue.js

```diff
@@ -243,25 +253,25 @@ export default class ObjectValue extends ConcreteValue {
   originalConstructor: void | ECMAScriptSourceFunctionValue;
 
   // partial objects
-  _isPartial: void | AbstractValue | BooleanValue;
+  _isPartial: AbstractValue | BooleanValue;
```

Comment:
I suppose this is what you mean by "cleaning up types". The reason why they were allowed to be void was to make them sparse. I.e. most objects would not be partial, so avoiding allocating a property entry for this case seemed like a good thing to me. Do you have data that suggests otherwise. Can you share it?

Reply:
Perhaps it would make more sense to remove those initializations?

Reply:
I expected to merely removing the initializations would do the trick. Perhaps I'm missing something, but it is really not all that obvious.